packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_controller.dart

```diff
@@ -161,6 +161,11 @@ class GoogleMapController {
     // Initial position can only to be set here!
     options = _applyInitialPosition(_initialCameraPosition, options);
 
+    // Fully disable 45 degree imagery if desired
+    if (options.rotateControl == false) {
+      options.tilt = 0;
```

Comment:
This might have unintended consequences now that `mapID` and possibly vector tiles are coming. But we'll see when that happens if this is a problem :)

Reply:
So are we leaving this as-is for now?

Reply:
Yes!